### ISSUE TITLE
Decimals in form field (with "number" type, no "text")

### ISSUE THREAD

**Issue by clemen2001** [elementor/elementor#11131]
## Prerequisites

- [ x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
hi. i´m spending hours looking for a solution.
i need to allow users at frontend place decimals numbers in a form.
now is only posible to place whole numbers, no decimals.
i have modified the class: **number.php** at plugins/elementor/includes/controls
but "step" it only works at backend, not at frontend.

protected function get_default_settings() {
return [
'min' => '',
'max' => '',
'step' => '0.01',
'placeholder' => '',
'title' => '',
'dynamic' => [
'categories' => [ TagsModule::NUMBER_CATEGORY ],

HELP

**Describe the solution you'd like**
modify the code to allow a STEP like 0,01 at backend and frontend.

**Describe alternatives you've considered**
i don´t know; i´m designer, not coder.

**Additional context**
example of the issue:
https://estancomayor49.com/number-issue/

THANKS!

**Comment by chrisb-TSS** [comment]
I agree that this would be a good update/addition to the multi step form Number field. The field is restricted to whole numbers for validation. 
Ideally, we would need a new field type added that is a "Currency" type but still uses a number validation.
Perhaps a rounding done to value after validating field ( is NaN ) then 
(+num).toFixed(2)

**Comment by Gruv69** [comment]
I'm having the same issue. Any solution?

Thanks

**Comment by JanineKall** [comment]
Same issue here, very annoying. I use 'Calculated Field for Elementor Form' to sum several fields, so they need to be numbers. When is this going to be solved?

**Comment by robaxx** [comment]
Just discovered this problem on my first Elementor form. What an insanity.
